src/System.Text.Json/System/Text/Json/JsonConstants.cs

```diff
@@ -30,9 +30,9 @@ static class JsonConstants
 
         #region Common values
 
-        public static readonly char[] TrueValue = { 't', 'r', 'u', 'e' };
-        public static readonly char[] FalseValue = { 'f', 'a', 'l', 's', 'e' };
-        public static readonly char[] NullValue = { 'n', 'u', 'l', 'l' };
+        public static readonly byte[] TrueValue = { (byte)'t', (byte)'r', (byte)'u', (byte)'e' };
+        public static readonly byte[] FalseValue = { (byte)'f', (byte)'a', (byte)'l', (byte)'s', (byte)'e' };
+        public static readonly byte[] NullValue = { (byte)'n', (byte)'u', (byte)'l', (byte)'l' };
```

Comment:
It seems like we are repeating these constants from above. Can we have just one copy?